### PR TITLE
Feature/panel desktop

### PR DESCRIPTION
**Pull request by msecret**
Check commit messages for explanations

### CODE REVIEW

**Comment by noahmanger** [comment]
This is looking really good. Not sure if you're still working on it, but the only thing I see worth noting is that the active row should be highlighted.

**Comment by msecret** [comment]
Yea I was still working on a few things like that. I also wanted to get the arrow on the rows done, move the close button on desktop.

What the status of those "View original image" button? Is there an existing link for that?

Also, for the perma-link, I'm guessing it will take about a day to implement that. Is it something I should do this next sprint, or are there other things that are more important?

**Comment by noahmanger** [comment]
Yeah I'd say it'd be good to do that this sprint. The "original image" link should exist...it may need to be built using the image numbers. @LindsayYoung how are we doing this now?

**Comment by noahmanger** [comment]
Regardless, I think we can merge this in whenever the active state is ready.

File: static/styles/_components/_panel.scss [hunk 1]
```diff
@@ -29,11 +43,35 @@
...
+    margin-top: 0;
+
+    &:last-child {
+      border-bottom: none;
+    }
+  }
+}
+
+#datatable-modal .panel .panel__row {
+  // TODO this shit is broken.
```
Comment (noahmanger) [inline static/styles/_components/_panel.scss]: What's broken?

Comment (msecret) [inline static/styles/_components/_panel.scss]: Haha, I was supposed to go back to that and write a better comment, I'll do so now

**Pull request merged**